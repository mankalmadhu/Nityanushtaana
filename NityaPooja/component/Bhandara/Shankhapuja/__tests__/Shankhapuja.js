import 'react-native';
import React from 'react';
import ಶ೦ಖಪೂಜಾ from '../Shankhapuja';

import Renderer from 'react-test-renderer';

jest.mock('../../../Avarana/Avarana',()=>'ಆವರಣ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');
jest.mock('../../../Vishaya/Purvanga',()=>'ಪೂರ್ವೋತ್ತರ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');
jest.mock('../../../Vishaya/Vidhana',()=>'ವಿಧಾನ');

test("ಶ೦ಖಪೂಜಾ" , () => {

expect(Renderer.create(
    <ಶ೦ಖಪೂಜಾ />
).toJSON()).toMatchSnapshot();

});