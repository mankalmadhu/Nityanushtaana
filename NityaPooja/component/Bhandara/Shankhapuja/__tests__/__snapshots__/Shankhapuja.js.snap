// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಶ೦ಖಪೂಜಾ 1`] = `
<ಆವರಣ>
  <ಶೀರ್ಷಿಕೆ>
    ಶ೦ಖಪೂಜಾ
  </ಶೀರ್ಷಿಕೆ>
  <ಪೂರ್ವೋತ್ತರ>
    ಕಲಶೋದಕೇನ ಶಂಖಂ ಪ್ರಕ್ಷಾಳ್ಯ । ಶಂಖಂ ಗಂಧಾಕ್ಷತ ಪತ್ರ ಪುಷ್ಪೈಃ ಅಭ್ಯರ್ಚ್ಯ । ಪ್ರಣವೇನ ಶಂಖಮಾಪೂರ್ಯ ॥
  </ಪೂರ್ವೋತ್ತರ>
  <ವಿಧಾನ>
    (ಕಲಶೋದಕದಿ೦ದ ಶ೦ಖವನ್ನು ತು೦ಬಿಸಿ, ಶ೦ಖವನ್ನು ಗ೦ಧ ಪುಷ್ಪದಿ೦ದ ಅಲ೦ಕರಿಸಿ, ಪ್ರಣವ ಮುದ್ರೆಯಲ್ಲಿ ಶ೦ಖವನ್ನು ಹಿಡಿದುಕೊಳ್ಳುವುದು.)
  </ವಿಧಾನ>
  <ವಿನಿಯೋಗ>
    ಶಂಖಂ ಚಂದ್ರಾರ್ಕದೈವತ್ಯಂ ವಾರುಣ೦ ಚಾಧಿದೈವತ೦ । ಪೃಷ್ಠೇ ಪ್ರಜಾಪತಿಸ್ತತ್ರ  ಅಗ್ರೇ ಗಂಗಾ ಸರಸ್ವತೀ ॥
  </ವಿನಿಯೋಗ>
  <ವಿನಿಯೋಗ>
    ತ್ರೈಲೋಕ್ಯೇ ಯಾನಿ ತೀರ್ಥಾನಿ ವಾಸುದೇವಸ್ಯ ಚಾಜ್ಞಯಾ । ಶಂಖೇ ತಿಷ್ಠಂತಿ ವಿಪ್ರೇ೦ದ್ರ ತಸ್ಮಾಚ್ಛಂಖಂ ಪ್ರಪೂಜಯೇತ್ ॥
  </ವಿನಿಯೋಗ>
  <ವಿನಿಯೋಗ>
    ತ್ವಂ ಪುರಾ ಸಾಗರೋತ್ಪನ್ನೋ ವಿಷ್ಣುನಾ ವಿಧೃತಃ ಕರೇ । ಪೂಜಿತಃ ಸರ್ವದೇವೈಶ್ಚ ಪಾಂಚಜನ್ಯ ನಮೋಽಸ್ತುತೇ ॥
  </ವಿನಿಯೋಗ>
  <ವಿನಿಯೋಗ>
    ಪಾಂ॒ಚ॒ಜ॒ನ್ಯಾಯ॑  ವಿ॒ದ್ಮಹೇ॑ ಪಾವಮಾ॒ನಾಯ॑ ಧೀಮಹಿ ।  ತನ್ನಃ॑ ಶಂಖಃ ಪ್ರಚೋ॒ದಯಾ᳚ತ್ ॥
  </ವಿನಿಯೋಗ>
  <ಪೂರ್ವೋತ್ತರ>
    ಶಂಖೋದಕಂ ಕಲಶೇ ಕಿ೦ಚಿನ್ನಿಕ್ಷಿಪ್ಯ । ಶ೦ಖೋದಕೇನ ಪೂಜಾದ್ರವ್ಯಾಣಿ ದೇವಂ ಆತ್ಮಾನಂಚ ಪ್ರೋಕ್ಷ್ಯ  । ಶೇಷಂ ವಿಸೃಜ್ಯ । ಪುನಃ ಶಂಖಮಾಪೂರ್ಯ ॥
  </ಪೂರ್ವೋತ್ತರ>
  <ವಿಧಾನ>
    (ಕಿ೦ಚಿನ್ ಶ೦ಖೋದಕವನ್ನು ಕಲಶೋದಕದಲ್ಲಿ ಮಿಶ್ರ ಮಾಡಿ , ಉಳಿದ ಶ೦ಖೋದಕವನ್ನು ಕ್ರಮವಾಗಿ ಪೂಜಾದ್ರವ್ಯ, ಪೂಜಾಮ೦ಟಪ ಹಾಗೂ ಸ್ವಯ೦ ಪ್ರೋಕ್ಷಣೆ ಮಾಡಿಕೊಳ್ಳುವುದು.ನ೦ತರ ಶ೦ಖವನ್ನು ಯಥಾಸ್ಥಾನದಲ್ಲಿ ಇರಿಸುವುದು. )
  </ವಿಧಾನ>
</ಆವರಣ>
`;
